src/include/sof/audio/kpb.h

```diff
@@ -12,6 +12,14 @@
 #include <user/trace.h>
 #include <stdint.h>
 
+#if defined(__XCC__)
+
+#include <xtensa/config/core-isa.h>
+#if XCHAL_HAVE_HIFI3 || XCHAL_HAVE_HIFI4
+#define KPB_HIFI4
```

Comment:
so this is HIFI3 actually, valid for TGL too, please rename

Reply:
Done